dask/dataframe/core.py

```diff
@@ -487,6 +484,13 @@ def _args(self):
         return (self.dask, self._name, self.name, self.divisions)
 
     @property
+    def _partition_type_sliced(self):
+        # return scalar
+        return lambda values, *args: values.item()
```

Comment:
Thanks, fixed to ``lambda v: df._partition_type(v).item()``, and removed unnecessary generalization as it only supports ``Series``.